scripts/network-init

```diff
@@ -108,6 +108,12 @@ rename_network_label() {
     done
 }
 
+maybe-enable-tls-verification() {
+  verif_enabled="$(${XE} pool-list params=tls-verification-enabled --minimal)"
+  if [ "$FIRSTBOOT_ENABLE_TLS_VERIFICATION" == "true" ] && [ "$verif_enabled" != "true" ] ; then
```

Comment:
I find `verif_enabled` a bit odd as a name. "I shot the verif, I didn't shoot the deputy?"